tutorials/tour/basics.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Running Scala on Browser
+
+You can run Scala on browser by using [ScalaFiddle](https://scalafiddle.io).
+
+1. Go to https://scalafiddle.io.
+2. Copy and paste `println("Hello, world!")` to the left pane.
+3. Hit "Run" button and see it prints "Hello, world!" on the right pane.
+
+It is a perfect way for anybody to experiment a piece of Scala code anywhere, anytime!
+
+## Variables
+
+You can define variables with the `val` keyword.
```

Comment:
Oh, I just didn't know the official term:) Thanks for pointing this out.

Maybe something like "You can define values (a.k.a. immutable variables) with the `val` keyword." will be correct and understandable? I see a lot of website referring `val` as immutable variables.